src/components/VAudioDetails/VRelatedAudio.vue

```diff
@@ -3,7 +3,10 @@
     <h4 class="mb-6 text-2xl lg:text-3xl">
       {{ $t('audio-details.related-audios') }}
     </h4>
-    <div v-if="!fetchState.isError" class="mb-12 flex flex-col gap-8 lg:gap-12">
+    <div
+      v-if="!fetchState.fetchingError"
```

Comment:
`.isError` property was removed some time in the past, but it wasn't caught without types.

Reply:
Good catch! What is stopping this file from being included in `tsconfig.json` file?

Reply:
I added it to `tsconfig.json` :)